commit 130513003054b93caaf82370aa990e8776aec3a5 (refs/changes/97/4197/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-13T17:39:39+00:00 (1 year, 4 months ago)
    
    MANTA-3757 pgstatsmon needs a registrar for waferlock
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index c9de41a..7c60df1 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1398,9 +1398,11 @@ cat <<EOF
     "public": true,
     "deps": [
       "amon",
-      "config-agent"
+      "config-agent",
+      "registrar"
     ],
     "tarballs": [
+      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
       {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
       {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
     ]
